packages/babel-plugin-transform-spread/test/fixtures/call-context/flow-type-cast/options.json

```diff
@@ -1,7 +1,10 @@
 {
-  "presets": [
+  "plugins": [
     [
-      "flow"
+      "transform-spread"
```

Comment:
Can you add plugin `external-helpers` here? So we don't have to update this test fixture when the helper code is changed.

Reply:
@JLHwung This doesn't seem to work for me. I get an output that doesn't have the spread transformation. Any ideas?

Reply:
Oh my bad. The `plugins` is overwritten completely.

```json
{
  "plugins": ["external-helpers", "transform-spread", "syntax-flow"]
}
```
should works.

Reply:
Resolved by https://github.com/babel/babel/pull/11404/commits/e6c3a4c5f0d381f2eeca881749efcd9eb56774db. Thanks for the tip!